documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,397 @@ keywords:
 ---
 import account_manipulate_did_rs from '!!raw-loader!../../../examples/account/manipulate_did.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
 
 
-TODO: Explain how DID Documents can be manipulated and how updates should be published.
+DID Documents can be extended by adding [Verification Methods](https://www.w3.org/TR/did-core/#verification-methods), [Services](https://www.w3.org/TR/did-core/#services) and custom properties.
+A verification method adds public keys, which can be used to digitally sign things like a DID message or a verifiable credential, while a service can provide metadata around the identity via URIs.
 
-## Example
+### Verification Methods
+As demonstrated by the [example](#example) below, the Iota identity framework offers easy-to-use methods for adding verification methods.
+
+The following properties can be specified for a verification method:
+
+- **id**: a [DID URL](https://www.w3.org/TR/did-core/#did-url-syntax) for the verification method. It can be specified by setting the [fragment](https://www.w3.org/TR/did-core/#fragment);
+- **type**: specifies the type of the Verification Method. The framework supports `Ed25519` and `X25519` key types. This property is automatically filled by the framework when specifying the verification material.
+- **publicKeyMultibase**: multibase encoded public key which concludes the [verification material](https://www.w3.org/TR/did-core/#verification-material). This can be automatically generated by the framework or manually provided by users.
+
+
+### [Verification Relationships](https://www.w3.org/TR/did-core/#verification-relationships)
```

Comment:
Maybe move the link away from the title and into the first mention of this term in the paragraph below?